apps/cookbook/src/app/examples/badge-example/examples/badge-example-small.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-badge-example-small',
+  template: `<kirby-badge size="sm">
+</kirby-badge>
```

Comment:
Consider adding elevation to these, and maybe a line about it to the showcase. The guidelines say: 

_"Badges are used without elevation unless they are white. White badges are used on avatars which make the badge disappear. Thus it has a z2 elevation."_

Reply:
I've added some elevation and a line about it :D